client/my-sites/marketing/connections/service-description.jsx

```diff
@@ -73,15 +73,15 @@ class SharingServiceDescription extends Component {
 			mailchimp: function () {
 				if ( this.props.numberOfConnections > 0 ) {
 					return this.props.translate(
-						'Allow users to sign up to your Mailchimp mailing list.',
-						'Allow users to sign up to your Mailchimp mailing lists.',
+						'Allow users to sign up to your Mailchimp mailing audience.',
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.